addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -115,19 +123,20 @@ var SnippetEditor = Widget.extend({
             // will decrease the animation counter after a fixed duration and
             // do the post animation cover if none is registered anymore.
             _animationsCount++;
+            this.$el.toggleClass('d-none', _animationsCount > 0);
             setTimeout(() => {
-                if (!--_animationsCount) {
+                if (_animationsCount > 0 && !--_animationsCount) {
                     postAnimationCover();
                 }
-            }, 500); // This delay have to be huge enough to take care of long
+            }, 1000); // This delay have to be huge enough to take care of long
                      // animations which will not trigger an animation end event
                      // but if it is too small for some, this is the job of the
                      // animation creator to manually ask for a re-cover
         });
         // On top of what is explained above, do the post animation cover for
         // each detected transition/animation end so that the user does not see
         // a flickering when not needed.
-        this.$target.on('transitionend.snippet_editor, animationend.snippet_editor', postAnimationCover);
```

Comment:
Don't remove the namespaces part otherwise, this is not `.off`'ed correctly in the destroy method

Reply:
Right!